modules/resolving-cuda-oom-errors.adoc

```diff
@@ -0,0 +1,65 @@
+:_module-type: PROCEDURE
+
+[id="resolving-cuda-oom-errors-for-the-single-model-serving-platform_{context}"]
+= Resolving CUDA out-of-memory errors
+
+[role="_abstract"]
+
+In certain cases, depending on the model and hardware accelerator used, the TGIS memory auto-tuning algorithm might underestimate the amount of GPU memory needed to process long sequences. This miscalculation can lead to Compute Unified Architecture (CUDA) out-of-memory (OOM) error responses from the model server. In such cases, you must update or add additional parameters in the TGIS model-serving runtime, as described in the following procedure.
+
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {odh-admin-group}) in {openshift-platform}.
+endif::[]
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the admin group (for example, {oai-admin-group}) in {openshift-platform}.
+endif::[]
+
+.Procedure
+. From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
++
+The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled.
++
+. Based on the runtime that you used to deploy your model, perform one of the following actions:
++
+* If you used the pre-installed *TGIS Standalone ServingRuntime for KServe* runtime, you must duplicate this runtime before following the remainder of this procedure. For more information about duplicating the pre-installed TGIS runtime, see {rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
```

Comment:
A few tweaks to my original suggested wording (sorry!):

```
If you used the pre-installed *TGIS Standalone ServingRuntime for KServe* runtime, duplicate the runtime to create a custom version and then follow the remainder of this procedure. To learn how to duplicate the pre-installed TGIS runtime, see {rhoaidocshome}{default-format-url}/serving_models/serving-large-models_serving-large-models#adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_serving-large-models[Adding a custom model-serving runtime for the single-model serving platform].
```

Reply:
Good suggestion! Updated in latest commit.